.github/workflows/dashboards-notifications-test-and-build-workflow.yml

```diff
@@ -156,7 +156,7 @@ jobs:
         run: |
           cd ./OpenSearch-Dashboards
           yarn start --no-base-path --no-watch --server.host="0.0.0.0" &
-          timeout 400 bash -c 'while [[ "$(curl -s http://localhost:5601/api/status | jq -r '.status.overall.state')" != "green" ]]; do echo sleeping 5; sleep 5; done'
+          timeout 900 bash -c 'while [[ "$(curl -s http://localhost:5601/api/status | jq -r '.status.overall.state')" != "green" ]]; do echo sleeping 5; sleep 5; done'
```

Comment:
Should it be 600?

Reply:
@Hailong-am , could you double check?

Reply:
I remember 600 is still not enough sometimes, so i increase the timeout to 900. Let's me change the description to 900s